src/field/pokemon.ts

```diff
@@ -4874,32 +4092,25 @@ export class PlayerPokemon extends Pokemon {
   addFriendship(friendship: number): void {
     if (friendship > 0) {
       const starterSpeciesId = this.species.getRootSpeciesId();
-      const fusionStarterSpeciesId = this.isFusion() && this.fusionSpecies ? this.fusionSpecies.getRootSpeciesId() : 0;
-      const starterData = [
-        globalScene.gameData.starterData[starterSpeciesId],
-        fusionStarterSpeciesId ? globalScene.gameData.starterData[fusionStarterSpeciesId] : null,
-      ].filter((d) => !!d);
+      const starterData = [globalScene.gameData.starterData[starterSpeciesId]].filter((d) => !!d);
       const amount = new NumberHolder(friendship);
       globalScene.applyModifier(PokemonFriendshipBoosterModifier, true, this, amount);
       let candyFriendshipMultiplier = CLASSIC_CANDY_FRIENDSHIP_MULTIPLIER;
       if (globalScene.eventManager.isEventActive()) {
         candyFriendshipMultiplier *= globalScene.eventManager.getFriendshipMultiplier();
       }
       const starterAmount = new NumberHolder(
-        Math.floor(
-          (amount.value * (globalScene.gameMode.isClassic ? candyFriendshipMultiplier : 1))
-            / (fusionStarterSpeciesId ? 2 : 1),
-        ),
+        Math.floor(amount.value * (globalScene.gameMode.isClassic ? candyFriendshipMultiplier : 1)),
       );
 
       // Add friendship to this PlayerPokemon
       this.friendship = Math.min(this.friendship + amount.value, 255);
       if (this.friendship === 255) {
         globalScene.validateAchv(achvs.MAX_FRIENDSHIP);
       }
-      // Add to candy progress for this mon's starter species and its fused species (if it has one)
-      starterData.forEach((sd: StarterDataEntry, i: number) => {
-        const speciesId = !i ? starterSpeciesId : (fusionStarterSpeciesId as Species);
+      // Add to candy progress for this mon's starter species
+      starterData.forEach((sd: StarterDataEntry) => {
+        const speciesId = starterSpeciesId;
         sd.friendship = (sd.friendship || 0) + starterAmount.value;
         if (sd.friendship >= getStarterValueFriendshipCap(speciesStarterCosts[speciesId])) {
           globalScene.gameData.addStarterCandy(getPokemonSpecies(speciesId), 1);
```

Comment:
```suggestion
      if (starterData) {
        starterData.friendship = (starterData.friendship || 0) + starterAmount.value;
        if (starterData.friendship >= getStarterValueFriendshipCap(speciesStarterCosts[starterSpeciesId])) {
          globalScene.gameData.addStarterCandy(getPokemonSpecies(speciesId), 1);
          starterData.friendship = 0;
```

** need to remove the sd.friendship = 0 line manually below since i couldn't grab it in the comment